File: XamarinRepairApp/XamarinRepairApp/Model/PropertyModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Json;

using Android.App;
using Android.Content;
using Android.OS;
using Android.Runtime;
using Android.Views;
using Android.Widget;
using XamarinRepairApp.Utils;

namespace XamarinRepairApp.Model
{
    public class PropertyModel
    {
        private JsonValue mData;

        public PropertyModel(JsonValue jsonValue)
        {
            mData = jsonValue;
        }

        public int GetId()
        {
            return Helper.GetInt(mData["ID"]);
        }

        public String GetTitle()
        {
            return Helper.GetString(mData["Title"]);
        }

        public String GetEmail()
        {
            return Helper.GetString(mData["sl_emailaddress"]);
        }

        public String GetOwner()
        {
            return Helper.GetString(mData["sl_owner"]);
        }

        public String GetAddress1()
        {
            return Helper.GetString(mData["sl_address1"]);
        }

        public String GetAddress2()
        {
            return Helper.GetString(mData["sl_address2"]);
        }

        public String GetCity()
        {
            return Helper.GetString(mData["sl_city"]);
        }

        public String GetState()
        {
            return Helper.GetString(mData["sl_state"]);
        }

        public String GetPostalCode()
        {
            return Helper.GetString(mData["sl_postalCode"]);
        }
    }
}

File: AndroidRepairApp/app/src/main/java/com/canviz/repairapp/data/PropertyModel.java
package com.canviz.repairapp.data;

import org.json.JSONException;
import org.json.JSONObject;

public class PropertyModel {
    private final JSONObject mData;

    public PropertyModel(JSONObject data) {
        mData = data;
    }

    public int getId(){
        try{
            return mData.getInt("ID");
        }catch(JSONException e) {
            return 0;
        }
    }

    public String getTitle(){
        try{
            return mData.getString("Title");
        }catch(JSONException e) {
            return null;
        }
    }

    public String getEmail(){
        try{
            return mData.getString("sl_emailaddress");
        }catch(JSONException e) {
            return null;
        }
    }

    public String getOwner(){
        try{
            return mData.getString("sl_owner");
        }catch(JSONException e) {
            return null;
        }
    }

    public String getAddress1(){
        try{
            return mData.getString("sl_address1");
        }catch(JSONException e) {
            return null;
        }
    }

    public String getAddress2(){
        try{
            return mData.getString("sl_address2");
        }catch(JSONException e) {
            return null;
        }
    }

    public String getCity(){
        try{
            return mData.getString("sl_city");
        }catch(JSONException e) {
            return null;
        }
    }

    public String getState(){
        try{
            return mData.getString("sl_state");
        }catch(JSONException e) {
            return null;
        }
    }

    public String getPostalCode(){
        try{
            return mData.getString("sl_postalCode");
        }catch(JSONException e) {
            return null;
        }
    }
}
